backends/bmv2/common/control.cpp

```diff
@@ -352,7 +352,9 @@ ControlConverter::convertTable(const CFG::TableNode* node,
             // 1) if there is at least one RANGE field, then the table is RANGE
             // 2) if there is at least one TERNARY or OPTIONAL field, then the table is TERNARY
             // 3) if there is a LPM field, then the table is LPM
-            // 4) otherwise the table is EXACT
+            // 4) if there is a LPM field, then the table is ternary or EXACT
+            // TODO: Do we also check if table is range or optional in 4?
```

Comment:
I don't understand this TODO. The rules above are pretty straightforward and just need to be applied.